dcalc/FindRoot.cc

```diff
@@ -17,6 +17,7 @@
 #include "FindRoot.hh"
 
 #include <algorithm> // abs
+#include <cstdlib> // abs for C++11 
```

Comment:
It looks to me like including just cmath makes more sense than including both cstdlib and algorithm.

Reply:
Good point, fixed.